src/Routing/Router.php

```diff
@@ -1010,11 +1010,12 @@ public static function parseRoutePath(string $url): array
         }
 
         $regex = '#^
-            (?:(?<plugin>[a-z0-9]+(?:/[a-z0-9]+)*)\.)?
-            (?:(?<prefix>[a-z0-9]+(?:/[a-z0-9]+)*)/)?
+            (?:(?<plugin>[a-z0-9]+(?:\/[a-z0-9]+)*)\.)?
+            (?:(?<prefix>[a-z0-9]+(?:\/[a-z0-9]+)*)\/)?
             (?<controller>[a-z0-9]+)
             ::
             (?<action>[a-z0-9_]+)
+            (?<params>[a-z0-9_\/]*)
```

Comment:
I don't think escaping the `/` is necessary since the regex uses `#` as delimiters not `/`.

Reply:
Ah.... so thats what this does at the start... Wasn't aware that one can set different regex delimiters 😅 
Will revert that back to how it was.

Reply:
Great idea 💡😫

Reply:
> Ah.... so thats what this does at the start... Wasn't aware that one can set different regex delimiters 😅 
> 
> Will revert that back to how it was.

What I think is 🤔.. Is gonna always gonna be problematic.. I have a Good ideas 💡 for it